openjdk-11.yaml

```diff
@@ -85,18 +85,28 @@ subpackages:
   - name: openjdk-11-jre
     pipeline:
       - runs: |
-          # generate JRE via jlink
-
-          mkdir -p ${{targets.subpkgdir}}/usr/lib/jvm
-          ${{targets.destdir}}/usr/lib/jvm/openjdk/bin/jlink \
-            --no-header-files \
-            --no-man-pages \
-            --compress=2 \
-            --add-modules ALL-MODULE-PATH \
-            --output ${{targets.subpkgdir}}/usr/lib/jvm/openjdk-jre
-
+          mkdir -p ${{targets.subpkgdir}}/usr/lib/jvm/openjdk
+          for dir in conf \
+                   include \
+                   jmods \
+                   legal \
+                   lib \
+                   release; do
+                      cp -r ./build/linux-*-server-release/images/jdk/${dir} ${{targets.subpkgdir}}/usr/lib/jvm/openjdk/${dir}
+          done
+          mkdir -p ${{targets.subpkgdir}}/usr/lib/jvm/openjdk/bin
+          for c in java \
+                  jjs \
+                  keytool \
+                  pack200 \
+                  rmid \
+                  rmiregistry \
+                  unpack200; do
+                      cp ./build/linux-*-server-release/images/jdk/bin/${c} ${{targets.subpkgdir}}/usr/lib/jvm/openjdk/bin/${c}
+          done
+    
           mkdir -p ${{targets.subpkgdir}}/usr/bin
-          ln -sf /usr/lib/jvm/openjdk-jre/bin/java ${{targets.subpkgdir}}/usr/bin/java
+          ln -sf /usr/lib/jvm/openjdk/bin/java ${{targets.subpkgdir}}/usr/bin/java
```

Comment:
looks good - you may already have this planned but in case not we'll need to update https://github.com/chainguard-images/images/blob/main/images/jre/configs/latest.apko.yaml#L29 to match the new symlink path after this merges